Tags style updated

gemdev111 opened the pull request:
Tags view updated style 
Integrated all tag
Update tags order all, trending then stablecoins
Integrated system images for stablecoins, trending, all

<table>
  <tr>
    <td><img src="https://github.com/user-attachments/assets/0ef29f05-59f2-4fb6-8f57-df0ae3c78a6e" style="width: 45%;" /></td>
        <td><img src="https://github.com/user-attachments/assets/39511bcf-4ee9-4beb-98e3-a7e5df3e19fc" style="width: 45%;" /></td>
  </tr>
</table>

closes #639

gemdev111 committed:
Tags view updates style
Update tags order trending then stablecoins
Integrated system images for stablecoins, trending

gemdev111 committed:
Updates assets tags

Added system images
Added all section

gemcoder21 commented on Packages/PrimitivesComponents/Sources/Components/TagsView.swift:
```diff
@@ -7,38 +7,73 @@ import Primitives
     }
 }
+
+public struct TagChip: View {
```
```suggestion
public struct TagChip: View {
```

Better name?

gemcoder21 commented on Packages/PrimitivesComponents/Sources/ViewModels/AssetTagViewModel.swift:
```diff
@@ -3,26 +3,45 @@
-    public var id: String { tag.rawValue }
-
-    public let tag: AssetTag
+    public let tag: AssetTag?
```
can we have non optional `AssetTag`?

gemcoder21 reviewed (changes requested):
Icons needs to be colorful, black/white does not fit here

gemdev111 committed:
Merge branch 'main'

gemdev111 committed:
Updates Tags

Removed images for tags
Created TagsView and protocols in components
Created AssetTagSelection to support all tag
Integrates TagsView in SelectAssetScene
Code cleanup

gemdev111 committed:
Merge branch 'main'

gemdev111 committed:
Removed useless AssetsTagsView prior to just TagsView

gemdev111 committed:
Merge branch 'main'

gemdev111 committed:
update core
